src/seismometer/configuration/config.py

```diff
@@ -275,6 +275,8 @@ def output_list(self) -> list:
 
         Configured in usage_data as outputs or primary_output.
         """
+        if self.output in self.usage.outputs:
```

Comment:
needs unit test, since this is a change in behavior, previously we would have had the output recorded twice

Reply:
FINC